fastlane/Fastfile

```diff
@@ -24,29 +24,34 @@ platform :android do
       else
         raise "Unknown flavor #{options[:flavor]}"
       end
+
     version_codes = google_play_track_version_codes(
       package_name: package_name,
       track: "internal",
     )
     last_version_code = version_codes.max
     next_version_code = last_version_code + 1
-    properties = {
+    properties = if options[:build_type] == "Debug"
+        {"android.injected.version.code" => next_version_code}
+    else
+        {
       "android.injected.signing.store.file" => ENV["KEYSTORE_FILE"],
       "android.injected.signing.store.password" => ENV["KEYSTORE_PASSWORD"],
```

Comment:
The entries in this hash literal should continue to be indented further than the curly braces surrounding them.